qtconsole/mainwindow.py

```diff
@@ -821,7 +821,9 @@ def set_syntax_style(self, syntax_style):
     get_ipython().run_line_magic('colors', '{colors}')
 """,
             True)
-        
+        self.active_frontend._append_plain_text(
+"""\n\nNote: Clearing the console after updating the syntax style
+is recommended for a better user experience.""")
```

Comment:
```suggestion
        self.active_frontend._append_plain_text(
            "\n\nNote: Clearing the console is necessary to fully apply the "
            "new syntax style you selected."
        )
```

Improve message and fix code style.